hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTable.java

```diff
@@ -258,11 +256,44 @@ protected void onBaseEntityEvent(final BaseUIEntityEvent<E> event) {
         return ids;
     }
 
+    /**
+     * Finds the entity object of the given entity ID by performing a database
+     * search
+     * 
+     * @param lastSelectedId
+     *            ID of the entity
+     * @return entity object as Optional
+     */
     protected abstract Optional<E> findEntityByTableValue(I lastSelectedId);
 
-    protected abstract void publishEntityAfterValueChange(E selectedLastEntity);
+    /**
+     * Is called inside the onValueChange()-method and is performed after
```

Comment:
Please adapt the comment as well:

1. That this method is called within 'onValueChange()-method' is not relevant and will be outdated soon
2. Don't tell the implementor what he has to do:
> If it is necessary to publish one or more events after selecting the entity, this method can be overridden   
It is just a callback that will be called after whenever a entity is selected